python/lsst/ap/association/assoc_db_sqlite.py

```diff
@@ -27,89 +27,25 @@
 
 from __future__ import absolute_import, division, print_function
 
-from collections import OrderedDict as oDict
 import numpy as np
 import sqlite3
 
 from lsst.meas.algorithms.indexerRegistry import IndexerRegistry
 import lsst.afw.table as afwTable
 import lsst.afw.geom as afwGeom
 import lsst.pex.config as pexConfig
-from lsst.pex.exceptions import InvalidParameterError
 import lsst.pipe.base as pipeBase
+from .assoc_afw_utils import \
+    make_minimal_dia_object_schema, \
+    make_minimal_dia_source_schema, \
+    ccdVisitSchema, \
+    add_dia_source_aliases_to_catalog, \
+    get_ccd_visit_info_from_exposure, \
+    make_overwrite_dict
```

Comment:
Why not just `import .afwUtils` (per my above filename comment), and then they'll all be namespaced. You'll have to expand that file's `__all__`.

Reply:
If you really don't want those things in the namespace, please make a note of that above the `__all__` in `afwUtils`, so that others don't expose it later.